### ISSUE TITLE
Github Action Broken

### ISSUE THREAD

**Issue by luigi311** [terra-money/terra.py#92]
- terra_sdk version:
- Python version:
- Operating System:

### Description

Looks like the github actions are broken again and the workflow has been changed around 4 times. Is there a reason the publish was fixed and reverted prior to a release? Looks like this commit 87fa46e60f852759708c6153dc3f04163aecd3a0 should of fixed all the issues since it added that --no-interaction flag but then it was removed in 9fa0c386ae3aa7f748f9c72d6302257a8d556f66 and the limit to only publish with 3.10 was removed in 1f4798f4b1fa3b25abc34c5571e886b48afdf090 

Was there a specific reason for this? I dont see a publish CI that was actually ran with the fixes before they were removed.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Vritra4** [comment]
actually it didn't work as we expected :(
not fixed yet becuase it's not urgent. i'll try later :)

**Comment by Vritra4** [comment]
fix: #149
